config/credentials.yml.enc

```diff
@@ -1 +1 @@
-PntLTrNBclUYNd89RpI3QDX55HIW7kFrpL5XVByoLwOhd0tImQxFI8gBUVfDUcatlWRZ70El24JibjYdY+Tnro5ZCsj7KCidWTP/+4Xw1vLZMB36ZVqLvZX/ohC+iIbLoyVLljj2qDS6GZyS6KhhfV6k8jqybgsmMsN6mSYzA6e/e6Qjfm0B6TOzJvi55tNdVZkjiOml/r8Zc+lgF0Ot1pmGkFn8tAFdM0noD6ArJedV8vvLZaTRdzWk9P9qZR3YF977ItiWJJwycW9to57ihowbcOrMrK7rkdR8pDJpL/Qa7c4JiMAgdjBDyQzkS7JvJyzR+rmL7GbjliatJJT+hLFUJfxTpe/+6v8K2RShbPX6V4CyyRrLADHwGkKXUO7Spkx5H8/cJ5QmveRoqGkfLKen75Qlj/02KTSW--jHe1+P/xgmQw1neI--mjijw3SrfamjLbW7jgcGyg==
\ No newline at end of file
+LJl5ArBauHWSlaFkBEyN9BshKOSM2f3fyAJW5M+d4zur3My6NedY+AdpNn37EVYLQyQTutBu/qT1rf3+8LK02GfUi7GqFJ6JnORPsGcH/leHhdz3b4lFOuRobUStnDsieetodlw6y0C6u1PJQWfzSLt9KnZ81AzfQcoFrl0+DKY0zzpw3+U5e1EleebU+bacigW963hLLPfwym5K5gaP6B0zD1Qvp9HzN9RIZECOEXS6qL7IahlpzrVGhEyIOulUrisZ3mGbDj3awd0vr9PNoMRpS6AHCSUP3gD/gcEEkv9+iBnQZSq0DYkvtUt/two8rlPY5RIWGDm9x/aVD0GqRYGuixwx/5vpm2QBtIM77bIUa8/AUaSa3SUSO++omPyp3qMV47B9zrpIrMDbyLCvMBk/KQALdEZ69eny3pxYuNA2ZRQAdtcvgveB/NmQwxRS5Il8rYNylnkoTWyNgnYf3sJMm6ge+HcqaEteIHQAmFiX+Bg4cKN8RqTpxVvude4uWe4seboGw6oDlOqqyoIKAon3Pr1XVGSZrR5I4Khs7JU=--rliYySD2LNDZ5KVc--58+gAOSepoH+yqKy0g66Sg==
```

Comment:
You should hide your private credentials.

Reply:
These are the encrypted ones. The key for decrypting them is placed in the .gitignore regardless. So this string is basically meaningless without the key.